### PR TITLE
Two themes

### PR DESCRIPTION
**Pull request by batchennings**
Proposition for two new themes. Added new fonts (_Lato_+_Spectral_) + overall work on ui consistency (typography, components, colors). Discreet animations on rollovers.
Stylesheet was also splitted into many scss parts. A "_variable.scss" file has been added in order to tweak colors in one place.

### CODE REVIEW

**Comment by Alkarex** [comment]
(Follow up of https://gitlab.com/patjennings/mapco/issues/1 and https://gitlab.com/patjennings/ansum/issues/1 )

@patjennings Many thanks for these great contributions! 
Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/dev/CREDITS.md
We will then merge in the /dev branch to potentially get additional feedback / testing :-)

**Comment by Alkarex** [comment]
One more thing from my side: the two themes use the same fonts (each time 0.6MB). Could you please move the fonts to https://github.com/FreshRSS/FreshRSS/tree/dev/p/themes/fonts/ to have only one copy?

**Comment by Alkarex** [comment]
Ups, the last commit https://github.com/FreshRSS/FreshRSS/pull/2245/commits/82566b2f24c703950407226ead69c669e6f9c66b erroneously changed access rights and other things in files under `./data/*` . I suggest you reset the last commit and re-apply only the relevant changes

**Comment by batchennings** [comment]
Oups, sorry... Mistake has been corrected. I took benefit of this update improving mobile theme.

**Comment by Alkarex** [comment]
@patjennings Great :-)
It looks like there is one remaining :
image

**Comment by batchennings** [comment]
Argh, this time i think it's ok... It seems that no `/data/` files are no longer part of this pull request.
But, this time, it seems that there are some checks on the travis build that are not completed, and i don't know exactly what it means : I'm afraid that problem is beyond my skills.
Let me know if you want me to make another clean PR from scratch.

**Comment by Alkarex** [comment]
Looks good. Let's do a bit more testing in the /dev branch :-)

File: p/themes/Ansum/_components.scss [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+	}
+    }
+    .separator {
+	margin: 0.75rem 0;
+	border-bottom: 1px solid $grey-light;
+	// display: none;
+    }
+
+}
+.tree .tree-folder .tree-folder-items .dropdown-menu,
```
Comment (Frenzie) [inline p/themes/Ansum/_components.scss:102]: @patjennings Is there a particular purpose to such duplicate selectors or is it just a minor oversight?

Comment (batchennings) [reply p/themes/Ansum/_components.scss:102]: well spotted. No particular reason for this.

**Pull request merged**